tempodb/encoding/common/interfaces.go

```diff
@@ -0,0 +1,53 @@
+package common
```

Comment:
+1 to this. let's go! this is what we need to make the big shift to a new block format.